src/backend/joanie/core/factories.py

```diff
@@ -150,6 +151,7 @@ class Meta:
     cover = factory.django.ImageField(
         filename="cover.png", format="png", width=1, height=1
     )
+    effort = factory.LazyFunction(lambda: timedelta(seconds=random.randint(1, 360000)))
```

Comment:
```suggestion
    effort = factory.Faker("time_delta", end_datetime=timedelta(hours=100))
```
I think it should work.

Reply:
It works perfectly. Thanks.